frontend/src/Teacher/AllTasks.vue

```diff
@@ -188,27 +187,22 @@ const onChangeSubject = () => {
 </script>
 
 <template>
-  <div v-if="!loaded" class="d-flex justify-content-center loading-animation">
-    <Loader />
+  <div class="d-flex gap-1 justify-content-start">
+    <select v-model="subject" class="form-select" @change="onChangeSubject">
+      <option value="all" selected>All</option>
+      <option v-for="subj in subjects" :key="subj.abbr" :value="subj.abbr">
+        {{ subj.name }}
+      </option>
+    </select>
   </div>
-  <template v-else>
-    <div class="d-flex gap-1 justify-content-start">
-      <select v-model="subject" class="form-select" @change="onChangeSubject">
-        <option value="all" selected>All</option>
-        <option v-for="subj in subjects" :key="subj.abbr" :value="subj.abbr">
-          {{ subj.name }}
-        </option>
-      </select>
-    </div>
-
-    <DataTable ref="dataTable" class="table table-striped" :columns="columns" :options="options">
-      <template #moss="props">
-        <a class="btn btn-secondary btn-sm" :href="`/teacher/task/${props.cellData.id}/moss`">
-          MOSS check
-        </a>
-      </template>
-    </DataTable>
-  </template>
+
+  <DataTable ref="dataTable" class="table table-striped" :columns="columns" :options="options">
+    <template #moss="props">
+      <a class="btn btn-secondary btn-sm" :href="`/teacher/task/${props.cellData.id}/moss`">
```

Comment:
This should be `/plagcheck` now, instead of `/moss` (this was an pre-existing bug). We should probably figure out a way of sharing routes between the backend and the frontend somehow.. something like https://github.com/tighten/ziggy.